package com.example.Test_task.util.exceptions;

public class PersonNotFoundException extends RuntimeException{

}
